src/components/Project/ProjectsAPI.js

```diff
@@ -0,0 +1,22 @@
+import axios from "axios";
+
+export default async function getProjects() {
+  let handlerError = false;
+  try {
+    const response = await axios.get(
+      "https://jsonplaceholder.typicode.com/users"
+    );
+    const result = await response.data;
```

Comment:
I don't think you need the await here
```
const { data: result } = await axios.get(
      "https://jsonplaceholder.typicode.com/users"
    );
```
You created an axios instance before I think that's why you had to use the double await